File: minwook/6주차/같은_숫자는_싫어.java
import java.util.*;

public class Solution {
    public int[] solution(int []arr) {
        List<Integer> list = new ArrayList<>();
        list.add(arr[0]);
        int n = arr[0];
        for(int i=0; i<arr.length; i++){
            if(n == arr[i]){
                n = arr[i];
                continue;
            }
            list.add(arr[i]);
            n = arr[i];
        }
        
        int[] array = list.stream().mapToInt(Integer::intValue).toArray();

        return array;
    }
}

File: jeonghyeon/6week/같은 숫자는 싫어.js
function solution(arr)
{
    var num = [];
    for(let i = 0; i < arr.length; i++) { // 현재 i부터 끝까지 반복
        if(arr[i] != arr[i + 1]) { // 현재와 그 다음수가 다른지 비교
            num.push(arr[i]); // 다르면 배열에 추가
        }
    }
    return num;
}